### ISSUE TITLE
They removed the deb files :)

### ISSUE THREAD

**Issue by uncomfyhalomacro** [flathub/com.microsoft.Teams#96]
I guess this is now the time to remove the package since I only see 404s now - https://packages.microsoft.com/repos/ms-teams/pool/main/t/teams/

![image](https://user-images.githubusercontent.com/66054069/220680096-29501a46-cdc7-402d-9f8f-552a6cbb3027.png)

**Comment by m5lapp** [comment]
As per [this issues for nixpkg](https://github.com/NixOS/nixpkgs/issues/217473#issuecomment-1443482974), it is still mirrored in the web archive. I don't know if it's worth having it hosted elsewhere though as [the PWA](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/microsoft-teams-progressive-web-app-now-available-on-linux/ba-p/3669846) seems to work quite nicely on Linux.

**Comment by daef** [comment]
the PWA works for you with screensharing and everything? @m5lapp

**Comment by m5lapp** [comment]
@daef , yep, not encountered any issues so far, was able to make a call and share my screen perfectly. I'm using Fedora 37 KDE spin. The outlook one also works great for emails/calendar etc.

**Comment by uncomfyhalomacro** [comment]
Yes the PWA works fine. I don't use it though since it's basically just a browser tab so I just open it in chromium or in firefox.

**Comment by daef** [comment]
I would have understood a PWA as something I open in my browser anyway.
